samples/ErrorHandlerSample/Startup.cs

```diff
@@ -22,7 +22,7 @@ public void Configure(IApplicationBuilder app)
                     await context.Response.WriteAsync("<html><body>\r\n");
                     await context.Response.WriteAsync("We're sorry, we encountered an un-expected issue with your application.<br>\r\n");
 
-                    var error = context.Features.Get<IErrorHandlerFeature>();
+                    var error = context.Features.Get<IExceptionHandlerFeature>();
```

Comment:
This rename is fine as I noticed that this type isn't being used any where in aspnet org.